src/Facets/GetGasFacet.sol

```diff
@@ -0,0 +1,102 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+import { ILiFi } from "../Interfaces/ILiFi.sol";
+import { ServiceFeeCollector } from "../Periphery/ServiceFeeCollector.sol";
+import { LibAsset, IERC20 } from "../Libraries/LibAsset.sol";
+import { LibDiamond } from "../Libraries/LibDiamond.sol";
+import { ReentrancyGuard } from "../Helpers/ReentrancyGuard.sol";
+import { InformationMismatch, InvalidConfig, AlreadyInitialized, NotInitialized } from "../Errors/GenericErrors.sol";
+import { SwapperV2, LibSwap } from "../Helpers/SwapperV2.sol";
+import { LibMappings } from "../Libraries/LibMappings.sol";
+import { Validatable } from "../Helpers/Validatable.sol";
+import { LibMappings } from "../Libraries/LibMappings.sol";
+
+/// @title GetGas Facet
+/// @author Li.Finance (https://li.finance)
+/// @notice Provides functionality for bridging gas through GetGas
+contract GetGasFacet is ILiFi, ReentrancyGuard, SwapperV2, Validatable {
+    
+    /// Storage ///
+    string internal constant FEE_COLLECTOR_NAME = "ServiceFeeCollector";
+
+    /// External Methods ///
+
+    /// @notice Bridges tokens via GetGas Bridge
+    /// @param _bridgeData Data used purely for tracking and analytics
+    function startBridgeTokensViaGetGas(
+        ILiFi.BridgeData memory _bridgeData
```

Comment:
won't work since the swap variation of the method modifies `_bridgeData` and both methods call into `_startBridge` so both have to use `memory` vs calldata.

Reply:
🚫